service/dap/server.go

```diff
@@ -1680,11 +1685,19 @@ func (s *Session) onContinueRequest(request *dap.ContinueRequest, allowNextState
 	s.runUntilStopAndNotify(api.Continue, allowNextStateChange)
 }
 
-func fnName(loc *proc.Location) string {
+func fnName(loc *proc.Location, hidePackagePaths bool) string {
 	if loc.Fn == nil {
 		return "???"
 	}
-	return loc.Fn.Name
+	name := loc.Fn.Name
+	if idx := strings.LastIndex(name, "/"); idx != -1 {
+		if hidePackagePaths {
+			return name[idx+1:]
+		} else {
+			return fmt.Sprintf("%s (in %s)", name[idx+1:], name[:idx])
```

Comment:
Could we make this be the whole package path (for `(in package/path)`)? so people don't have to refer back to the start of the string for the full path.

Reply:
Yep, fixed in ca8e87194a50. (Let me know if you prefer amending and force-pushing over fixup commits, not sure what the conventions around here are.)